src/App.js

```diff
@@ -1,15 +1,12 @@
 import React, {Component} from "react";
-import NavBar from './components/NavBar';
+import Main from './components/main';
+
 import './App.css'
 
 class App extends Component {
```

Comment:
change to functional component

if the component will be stateless you can remove the return part and jest do 
```export default ()=> ( <Main />)```